package.json

```
@@ -43,6 +43,7 @@
     "@typescript-eslint/eslint-plugin": "2.19.0",
     "@typescript-eslint/parser": "2.19.0",
     "babel-eslint": "10.1.0",
+    "babel-plugin-inline-replace-variables": "^1.3.1",
```

Dev dependencies are pinned (and need to re-run `yarn install`).

```suggestion
    "babel-plugin-inline-replace-variables": "1.3.1",
```

---

@SophieManley03 If you applied from this suggestion you need to re-reun `yarn` locally to re-generate `yarn.lock`. This is why your build is currently failing.